samples/boards/nrf52/mesh/onoff_level_app/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: Bluetooth Mesh
+tests:
+  test:
+    harness: bluetooth
+    platform_whitelist: bbc_microbit nrf51_blenano qemu_x86
```

Comment:
This sample is specifically for nRF52, so the above whitelist looks wrong to me. It should only contain one or more nRF52 targets. The same applies for the various *.conf files for microbit, etc. There should only be a single prj.conf that targets the nRF52.